client/src/app/site/agenda/components/agenda-list/agenda-list.component.html

```diff
@@ -171,6 +171,12 @@
             </button>
             <mat-divider></mat-divider>
 
+            <!-- Duplicate items -->
+            <button mat-menu-item [disabled]="!selectedRows.length" (click)="duplicateMultipleTopics(selectedRows)">
+                <mat-icon>file_copy</mat-icon>
+                <span translate>Duplicate</span>
```

Comment:
Not sure if "duplicate" is the best wording we can find here. Google drive offers a similar feature, which they call "create copy"